src/Standards/Generic/Sniffs/ControlStructures/DisallowYodaConditionsSniff.php

```diff
@@ -86,15 +81,15 @@ public function process(File $phpcsFile, $stackPtr)
         // Is it a parenthesis.
         if ($tokens[$previousIndex]['code'] === T_CLOSE_PARENTHESIS) {
             // Check what exists inside the parenthesis.
```

Comment:
Seeing the variable rename, I wonder is this comment is correct ?

Reply:
I read this comment as referring to everything that is inside the if statement and not only to what is happening in the line below. But even considering this, the comment is not correct. The code mostly checks what is inside the parenthesis, but it also returns without checking what is inside the parenthesis if it belongs to a function call. So I went ahead and removed it.